package.json

```diff
@@ -33,5 +33,8 @@
     "ts-jest": "^18.0.3",
     "tslint": "^3.15.1",
     "typescript": "^2.1.5"
+  },
```

Comment:
I think we can clean the package.json up a bunch. Kill the unused deps but you will also need to do name and description